### PR TITLE
testing SortedList

### PR DESCRIPTION
**Pull request by SargisV**

### CODE REVIEW

File: homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <dependencies>
+<!--        <dependency>-->
+<!--            <groupId>junit</groupId>-->
+<!--            <artifactId>junit</artifactId>-->
+<!--            <version>4.13.2</version>-->
+<!--            <scope>test</scope>-->
+<!--        </dependency>-->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
```
Comment (sirak92) [inline homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml:20]: only having this dependency isn't enough, if you check maven test will not run unit tests. You need to include junit-jupiter artificat id for that

File: homeworks/sargis_vardanyan/HelloWorldMaven/target/maven-archiver/pom.properties [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```
Comment (sirak92) [inline homeworks/sargis_vardanyan/HelloWorldMaven/target/maven-archiver/pom.properties:1]: the target folder is autogenerated one, and should be added to ignored list in .gitignore

File: homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml (hunk 1 above)
Comment (SargisV) [reply homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml:20]: I got that question, about why mvn test command show the this output(0 tests.....), but working with Intellij IDEA's tools everything works as it should be, and also what about <build></build>, and <plugins></plugins>, do I need them for this task, and if yes why?

Comment (sirak92) [reply homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml:20]: I'm not sure if I get you right, but Intellij runs the tests itself, it doesn't need maven plugin to run it, but the maven needs it

File: homeworks/sargis_vardanyan/HelloWorldMaven/target/maven-archiver/pom.properties (hunk 2 above)
Comment (SargisV) [reply homeworks/sargis_vardanyan/HelloWorldMaven/target/maven-archiver/pom.properties:1]: after changing .gitignore as mentioned, I tried 
`git add .` 
and then, commit, push, but the target folder still exists

File: homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml (hunk 1 above)
Comment (sirak92) [reply homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml:20]: first of all, intellij doesn't use maven to build the maven project (although it still parse pom.xml to find the dependencies, repos etc..). The plugin for maven is the part that responsible for doing real actions like compliation, jar creation, packaging, deploying etc (https://maven.apache.org/guides/introduction/introduction-to-plugins.html#:~:text=What%20is%20a%20Plugin%3F,documentation%2C%20and%20on%20and%20on.)
So maven plugin is responsible for running unit tests, so if you skip the plugin, the unit tests can be running from the intellij but not from the maven

Comment (SargisV) [reply homeworks/sargis_vardanyan/HelloWorldMaven/pom.xml:20]: Thanks a lot

**Pull request merged**